assets/dev/js/frontend/tools/stretch-element.js

```diff
@@ -34,9 +35,10 @@ module.exports = elementorModules.ViewModule.extend( {
 			containerWidth = $container.innerWidth(),
 			elementOffset = $element.offset().left,
 			isFixed = 'fixed' === $element.css( 'position' ),
-			correctOffset = isFixed ? 0 : elementOffset;
+			correctOffset = isFixed ? 0 : elementOffset,
+			isContainerFullScreen = window === $container[ 0 ];
```

Comment:
if the above `try` is for jQuery not existing, then container will not be an array - will be undefined, so accessing `$container[ 0 ]` will not work.

Reply:
That's what line 29 is for, it will always be a jQuery instance.

Reply:
cool, cool cool